ipamd/ipamd.go

```diff
@@ -295,14 +295,21 @@ func (c *IPAMContext) nodeInit() error {
 	}
 	c.updateIPStats(numUnmanaged)
 
+	var pbVPCcidrs []string
+	vpcCIDRs := c.awsClient.GetVPCIPv4CIDRs()
+
+	for _, cidr := range vpcCIDRs {
+		pbVPCcidrs = append(pbVPCcidrs, *cidr)
+	}
+
 	_, vpcCIDR, err := net.ParseCIDR(c.awsClient.GetVPCIPv4CIDR())
```

Comment:
`GetVPCIPv4CIDR` is called twice here, we should rely on the fact that it's cached in c.awsClient